frontend/dist/index.html

```diff
@@ -64,8 +64,8 @@
 <body>
 <div id='content'></div>
 
-<script type='text/javascript' src='./app/tracim_frontend_vendors.js'></script>
-<script type='text/javascript' src='./app/tracim_frontend_lib.lib.js'></script>
+<script type='text/javascript' src='/app/tracim_frontend_vendors.js'></script>
```

Comment:
does this change fixes something is it just for consistency ?

Reply:
This is just for consistency